widget/entry_cursor_anim.go

```diff
@@ -0,0 +1,197 @@
+package widget
+
+import (
+	"image/color"
+	"sync"
+	"time"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/canvas"
+	"fyne.io/fyne/v2/theme"
+)
+
+// ===============================================================
+// Cursor ticker
+// ===============================================================
+
+type cursorTicker struct {
+	tim      *time.Timer
+	rstCh    chan struct{}
+	duration time.Duration
+	mockWait func() (reset bool)
+}
+
+func (t *cursorTicker) Start() {
```

Comment:
Yes, my mistake, I will fix it.

Reply:
> Also the ordering - Start should come after Reset to keep it alphabetical

Oh ok, I thought it would be better to order the functions according to the order they are intended to be called.

Reply:
done